docs/start/getting-started/fragments/flutter/integrate.md

```diff
@@ -1,120 +1,470 @@
-In this tutorial, you will integrate basic functionality for **Analytics**.
+In this section you’ll integrate Amplify DataStore with your app, and learn to use the generated data model to create, update, query, and delete Todo items by building an app. You can find the source code of the final Todo App [on our GitHub repository](https://github.com/cshfang/amplify-flutter-datastore-starter/tree/main/amplified_todo).
 
-First, delete the contents of your app's *main.dart* file and paste in this starter boilerplate UI code.   
+First, replace the contents of your *main.dart* file with the following UI boilerplate code. Typically, you would break this file up into smaller modules but we've kept it as a single file here just for the tutorial. You might find your IDE complains about numerous unreferenced declarations but, don’t worry, we’ll get around to fixing those as we fill out our app with more functionality.
 
 ```dart
+// dart async library we will refer to when setting up real time updates
+import 'dart:async';
+// flutter and ui libraries
 import 'package:flutter/material.dart';
+// amplify packages we will need to use
 import 'package:amplify_flutter/amplify.dart';
-import 'package:amplify_analytics_pinpoint/amplify_analytics_pinpoint.dart';
-import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_datastore/amplify_datastore.dart';
+// amplify configuration and models that should have been generated for you
 import 'amplifyconfiguration.dart';
+import 'models/ModelProvider.dart';
+import 'models/Todo.dart';
```

Comment:
Ah I wasn't aware we had collapsible segments. It would be nice if we could truncate some of these large code blocks where the display allows expansion but the copy still copies the whole snippet... do you know if that is possible today with the collapsible segments you alluded to?

Fwiw, I do think that the outline navigation allows for some skipping around as is but I can definitely see the initial boilerplate being a big of a deterrent (though I really did try to go through and trim the fat!)

Reply:
to be honest it's more of a wishlist at this point. CC: @tjleing @wlee221 to see what's possible here.

Would breaking this page down in multiple pages help?
* Create a boilerplate flutter app
* Add Local DataStore
* Add online sync